client/src/app/views/profile/views/profile-invoices/profile-invoices.component.html

```diff
@@ -42,7 +42,11 @@
 					<td>{{invoice.date | date:"d. M. y"}}</td>
 					<td class="small text-muted"><ng-container *ngIf="invoice.paragraph">§{{invoice.paragraph}}</ng-container></td>
 					<td class="small text-muted"><ng-container *ngIf="invoice.item">#{{invoice.item}}</ng-container></td>
-					<td><a *ngIf="invoice.counterpartyId" [href]="'http://wwwinfo.mfcr.cz/cgi-bin/ares/ares_es.cgi?jazyk=cz&ico=' + (invoice.counterpartyId | ico) + '&cestina=cestina&xml=1'" target="_blank">{{invoice.counterpartyId | ico}}</a></td>
+					<td>
+						{{invoice.counterpartyId | ico}}
+						<a *ngIf="invoice.counterpartyId" [href]="'http://wwwinfo.mfcr.cz/cgi-bin/ares/ares_es.cgi?jazyk=cz&ico=' + (invoice.counterpartyId | ico) + '&cestina=cestina&xml=1'" target="_blank">[Ares]</a>
+						<a *ngIf="ico" [href]="'https://www.hlidacstatu.cz/hledat?Q=ico%3A' + (ico)" target="_blank">[Hlídač státu]</a>

```

Comment:
Tady by mělo být asi spíš invoice.counterpartyId místo ico ;) Případně můžeš použít Pipe ico jako je výše, ale to jen doplní nuly dopředu (viz https://github.com/cityvizor/cityvizor/blob/d5406c07ffb51093c4a07ee31512b2dc02eff0db/client/src/app/shared/pipes/utils.pipe.ts#L14)

Reply:
Ok, vyzkouším díky.